.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish package on NPM
+on:
+  release:
+    types: [published]
```

Comment:
in https://github.com/DataDog/build-plugin/blob/master/.github/workflows/publish.yaml#L4 we have `[created]` but I think we this we avoid creating releases for drafts. This worked quite well for us at scope: https://github.com/DataDog/scope-javascript-agent/blob/master/.github/workflows/npmpublish.yml#L5

Reply:
Oh, that's interesting.

Reply:
sadly the docs around this are not great 😆 https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onevent_nametypes